internal/pkg/db/mongo/export-mongo-client.go

```diff
@@ -32,13 +34,13 @@ Export client client
 Has functions for interacting with the export client mongo database
 */
 
-type MongoClient struct {
+type ExportMongoClient struct {
 	Session  *mgo.Session  // Mongo database session
 	Database *mgo.Database // Mongo database
 }
 
-// Return a pointer to the MongoClient
-func newMongoClient(config DBConfiguration) (*MongoClient, error) {
+// Return a pointer to the ExportMongoClient
+func NewExportMongoClient(config db.Configuration) (*ExportMongoClient, error) {
```

Comment:
This can be removed and use NewClient() and Connect() defined in internal/pkg/db/mongo/client.go